ci/terraform-taint/meta-terraform-taint.yml

```
@@ -0,0 +1,37 @@
+meta-terraform-taint:
+  plan:
+    terraform-taint:
+      task: terraform-taint
+      config:
+        platform: linux
+        image_resource:
+          type: docker-image
+          source:
+            repository: ((terraform.repository))
+            version: ((terraform12.version))
+            tag: ((terraform12.version))
+        inputs:
+          - name: tfrepo
+        params:
+          TF_CLI_ARGS_plan: -lock-timeout=300s
+          TF_INPUT: "false"
+          TF_VAR_slack_webhook_url: ((slack.webhook_url))
+          AWS_ACCESS_KEY_ID: ((ci.aws_access_key_id))
+          AWS_SECRET_ACCESS_KEY: ((ci.aws_secret_access_key))
+          RESOURCE_ADDRESS_LIST: "unset unset"
+        run:
+          path: sh
+          dir: tfrepo
```

I found this confusing
```suggestion
          dir: tfrepo
          path: sh
```

---

Yeah, true, changed.